content/concepts/protocols.md

```diff
@@ -1,227 +1,154 @@
-the matching logic described above to accept or reject the protocol. If the protocol is rejected, the dialing peer can try again.
+When opening a new stream on a connection, the initiating peer sends the protocol 
+ID it would like to use. The remote peer will use the matching logic described above to 
+accept or reject the protocol. If the protocol is rejected, the dialing peer can try again.
 
-When dialing, you can optionally provide a list of protocol ids instead of a single id. When you provide multiple protocol ids, they will
-each be tried in succession, and the first successful match will be used if at least one of the protocols is supported by the remote peer.
-This can be useful if you support a range of protocol versions, since you can propose the most recent version and fallback to older versions
-if the remote hasn't adopted the latest version yet.
+When opening a new stream, the initiator can optionally provide a list of protocol IDs instead 
+of a single ID. When you provide multiple protocol IDs, they will each is tried in succession, 
+and the first successful match will be used if at least one of the protocols is supported by the 
+remote peer. This can be useful if you support a range of protocol versions since you can 
+propose the most recent version and fallback to older versions if the remote hasn't adopted 
+the latest version yet.
 
 ## Core libp2p protocols
 
-In addition to the protocols that you write when developing a libp2p application, libp2p itself defines several foundational protocols that are used for core features.
+In addition to the protocols written when developing a libp2p application, libp2p defines 
+several foundational protocols used for core features.
 
 ### Common patterns
 
-The protocols described below all use [protocol buffers](https://developers.google.com/protocol-buffers/) (aka protobuf) to define message schemas.
-
-Messages are exchanged over the wire using a very simple convention which prefixes binary
-message payloads with an integer that represents the length of the payload in bytes. The
-length is encoded as a [protobuf varint](https://developers.google.com/protocol-buffers/docs/encoding#varints)  (variable-length integer).
-
-
-
-### Ping
-
-| **Protocol id**    | spec |               |               | implementations   |
-|--------------------|------|---------------|---------------|-------------------|
-| `/ipfs/ping/1.0.0` | N/A  | [go][ping_go] | [js][ping_js] | [rust][ping_rust] |
-
-
-[ping_go]: https://github.com/libp2p/go-libp2p/tree/master/p2p/protocol/ping
-[ping_js]: https://github.com/libp2p/js-libp2p-ping
-[ping_rust]: https://github.com/libp2p/rust-libp2p/blob/master/protocols/ping/src/lib.rs
-
-
-The ping protocol is a simple liveness check that peers can use to quickly see if another peer is online.
+The protocols described below all use 
+[protocol buffers](https://developers.google.com/protocol-buffers/) 
+(aka protobuf) to define message schemas.
```

Comment:
This should be mentioned above, where we say that the application protocol defines all the bytes after the multistream negotiation. And this statement should be weaker, something like "a lot of libp2p protocols use protobufs".